src/main/java/com/google/codeu/data/Location.java

```diff
@@ -34,7 +34,7 @@
    * random ID and uses the current system time for the creation time.
    */
   public Location(String owner, String title, String description, long latitude, long longitude) {
-    this(UUID.randomUUID(), owner, title, latitude, longitude, System.currentTimeMillis());
+    this(UUID.randomUUID(), owner, title, description, latitude, longitude, System.currentTimeMillis());
```

Comment:
we might've patched this up recently already i think? might cause merge conflicts, otherwise thanks for fixing!!